pureskillgg_dsdk/adx/writer_fs.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+import urllib
+
+import structlog
+
+
+class AdxDatasetWriterFs:
+    def __init__(self, *, root_path, prefix, log=None):
+        self._log = log if log is not None else structlog.get_logger()
+        self._log = self._log.bind(client="adx_dataset_writer_fs")
+        self._path = root_path
+        self._prefix = prefix
+
+    def export_revision(self, client, dataset_id, revision_id):
+        assets = get_assets(client, dataset_id, revision_id)
+        if len(assets) > 500:
+            self._log.warn(
+                f"This revision contains {len(assets)} assets and will take a long time to download. Seriously, you are not gonna want to wait for this. Export to S3 instead."
```

Comment:
```suggestion
                f"This revision contains {len(assets)} assets and will take a long time to download. Seriously, you are not gonna want to wait for this. Export to S3 by using export_multiple_adx_dataset_revisions_to_s3 instead."
```
optional but i think it is nice to be specific.